api/api/src/main/scala/uk/ac/wellcome/platform/api/models/MustQuery.scala

```diff
@@ -1,6 +1,7 @@
 package uk.ac.wellcome.platform.api.models
 
-sealed trait MustQuery
-sealed trait ImageMustQuery extends MustQuery
+sealed trait WorkMustQuery
```

Comment:
Where is this trait used? I see it in a bunch of function signatures, but I don’t see an example of something that actually extends it (and surely that would have to be in this file, since it's a sealed trait).

Reply:
We need to provide two type parameters for this:

```scala
case class SearchOptions[DocumentFilter, MustQuery](
  ...
)
```

The `MustQuery` type is used for images and not for works. but we still need to provide something. I added a new type for this, as imo that is much clearer to read than passing in `Unit` or something everywhere, and in future we may actually extend from it (there are a number of other options here such as a type alias, or using an empty `object` type, but both also have downsides too)